cupcakescodersDRF/users/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+from .models import CustomUser
+
+class CustomUserSerializer(serializers.Serializer):
+    id = serializers.ReadOnlyField()
+    student_id = serializers.CharField(max_length=100)
+    date_created = serializers.ReadOnlyField()    
+    first_name = serializers.CharField(max_length=100)
+    last_name = serializers.CharField(max_length=100)
+    email = serializers.EmailField()    
+
+    def create(self, validated_data):
+        return CustomUser.objects.create(**validated_data)
```

Comment:
I think this should use objects.create_user model? otherwise the password will not be encrypted.